runtime/compiler/codegen/J9AheadOfTimeCompile.hpp

```diff
@@ -57,17 +57,26 @@ class OMR_EXTENSIBLE AheadOfTimeCompile : public OMR::AheadOfTimeCompileConnecto
    uint8_t* dumpRelocationHeaderData(uint8_t *cursor, bool isVerbose);
 
    /**
-    * @brief Initializes the common fields in the raw relocation record header. It then delegates
-    *        to initializePlatformSpecificAOTRelocationHeader which must be implemented on all
-    *        platforms that support AOT.
+    * @brief Initializes the common fields in the raw relocation record header.
```

Comment:
I think this method initializes the entire relocation record header

Reply:
Done in [force push](https://github.com/eclipse-openj9/openj9/compare/6a8bd2a1f7ba7914cf66c88fca7ab123435c12fc..3ae2f331966f06beab38c71248952813bcac3f04)